src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.DefaultGlobalUsing.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NET6_0
```

Comment:
```suggestion
#if NET6_0_OR_GREATER
```

Reply:
🤦

Reply:
Oops, wrong comment, the facepalm is for the other typo fix.

Reply:
Super cool! I haven't seen this feature used anywhere yet: https://github.com/dotnet/designs/blob/main/accepted/2020/or-greater-defines/or-greater-defines.md.